stacks/core/dev/stage-1.yml

```
@@ -41,6 +41,20 @@ services:
         - traefik.port=8183
 
   # KNoT Fog Core
+  storage:
+    image: cesarbr/knot-cloud-storage-go:dev
+    env_file: './env.d/knot-cloud-storage.env'
+    volumes:
+      - ../knot-cloud-storage:/usr/src/app
+    depends_on:
+      - mongo
+    deploy:
+      replicas: 1
+      labels:
+        - traefik.enable=true
+        - traefik.frontend.rule=HostRegexp:data,data.{domain:[a-zA-Z0-9.]+}
```

Personally, I believe the traefik rule should be representative of the services name :point_down: 
```suggestion
        - traefik.frontend.rule=HostRegexp:storage,storage.{domain:[a-zA-Z0-9.]+}
```

---

I agree, what @joaoaneto think about it?

---

I agree with you!

---

Done! :smile: